<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoCardDialog</class>
 <widget class="QDialog" name="NoCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget_head" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QToolButton" name="toolButton_exit">
    <property name="geometry">
     <rect>
      <x>958</x>
      <y>19</y>
      <width>44</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_switch">
    <property name="geometry">
     <rect>
      <x>766</x>
      <y>19</y>
      <width>44</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>25</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>通道选择</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_chanel">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>23</y>
      <width>111</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_shutdown">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>19</y>
      <width>44</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_opengate">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>19</y>
      <width>44</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_restart">
    <property name="geometry">
     <rect>
      <x>894</x>
      <y>19</y>
      <width>44</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>25</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>手动开闸</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_curtime">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>31</y>
      <width>231</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_opengate_2">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>19</y>
      <width>44</width>
      <height>43</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_nocard_main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>1024</width>
     <height>688</height>
    </rect>
   </property>
   <widget class="QWidget" name="widget_page_switch_nocard" native="true">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>0</y>
      <width>980</width>
      <height>666</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_nocard_content">
     <property name="geometry">
      <rect>
       <x>312</x>
       <y>10</y>
       <width>400</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>无证人员信息</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_gate">
     <property name="geometry">
      <rect>
       <x>880</x>
       <y>10</y>
       <width>50</width>
       <height>48</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_pic1">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>100</y>
       <width>145</width>
       <height>178</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_pic2">
     <property name="geometry">
      <rect>
       <x>417</x>
       <y>100</y>
       <width>145</width>
       <height>178</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_pic3">
     <property name="geometry">
      <rect>
       <x>698</x>
       <y>100</y>
       <width>145</width>
       <height>178</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_static1">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>337</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>姓    名:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_static2">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>378</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>性    别:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_static3">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>419</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>民    族:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_static4">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>460</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>出生年月:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_name">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>337</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_sex">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>378</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_nation">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>419</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_birthday">
     <property name="geometry">
      <rect>
       <x>222</x>
       <y>460</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_static5">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>337</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>住    址:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_static6">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>378</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>身份证号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_static7">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>419</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>签发机关:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_static8">
     <property name="geometry">
      <rect>
       <x>465</x>
       <y>460</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>有 效 期:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_address">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>337</y>
       <width>420</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_cardno">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>378</y>
       <width>420</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_department">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>419</y>
       <width>420</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_nocard_date">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>460</y>
       <width>420</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_nocard_deal">
     <property name="geometry">
      <rect>
       <x>644</x>
       <y>560</y>
       <width>200</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>处理完成</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_nocard_verifyno">
     <property name="geometry">
      <rect>
       <x>136</x>
       <y>560</y>
       <width>200</width>
       <height>47</height>
      </rect>
     </property>
     <property name="text">
      <string>人脸比对</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
